esphome/components/sen21231/sensor.py

```diff
@@ -0,0 +1,33 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.components import i2c, sensor
+from esphome.const import CONF_ID, ICON_MOTION_SENSOR, UNIT_EMPTY
+
+DEPENDENCIES = ["i2c"]
+
+CONF_I2C_ADDR = 0x62
+
+sen21231_sensor_ns = cg.esphome_ns.namespace("sen21231_sensor")
+Sen21231Sensor = sen21231_sensor_ns.class_(
+    "Sen21231Sensor", cg.PollingComponent, i2c.I2CDevice
+)
+
+CONFIG_SCHEMA = (
+    sensor.sensor_schema(
+        unit_of_measurement=UNIT_EMPTY, icon=ICON_MOTION_SENSOR, accuracy_decimals=1
+    )
+    .extend(
+        {
+            cv.GenerateID(): cv.declare_id(Sen21231Sensor),
+        }
+    )
+    .extend(cv.polling_component_schema("30s"))
+    .extend(i2c.i2c_device_schema(CONF_I2C_ADDR))
```

Comment:
No need for a new CONST for the address
```suggestion
    .extend(i2c.i2c_device_schema(0x62))
```